packages/upgradeable/src/providers/validation.ts

```diff
@@ -60,18 +140,15 @@ export class ValidationProvider {
    * @param artifactName Name of the compiled contract.
    * @param deploymentName Name of the deployed contract. Consist from the actual contract name and tags.
    */
-  private async validateUpgrade(artifactName: string, deploymentName: string, options: StandaloneOptions) {
+  private async validateUpgrade(artifactName: string, deploymentName: string) {
     const deployment = await this.hre.deployments.getOrNull(deploymentName)
     if (!deployment) {
-      this.logger.log(`No deployment record found for "${deploymentName}". Has the contract never been deployed?`)
+      this.logger.debug(`No deployment record found for "${deploymentName}". Has the contract never been deployed?`)
       return
     }
```

Comment:
`validateDataInSync` and this method have same logic, maybe moe it in parent function?

Reply:
this functions might be used separately later, so just for sanity check i'll leave it here (maybe i fix it later, once lib is done)